cosmos-gui/src/cmd_runner.js

```diff
@@ -28,18 +28,19 @@ var spawn = require('child_process').spawn;
 
 function run(cmd, params, callback) {
     var execution = spawn(cmd, params);
-    var result = '';
+    var resStdout = '';
+    var resStderr = '';
 
     execution.stdout.on('data', function (data) {
-        result += 'stdout: ' + data.toString();
+        resStdout += data.toString();
     });
 
     execution.stderr.on('data', function (data) {
-        result += 'stderr: ' + data.toString();
+        resStderr += + data.toString();
     });
 
     execution.on('close', function (code) {
-        return callback(null, result);
+        return callback(null, resStdout.length == 0 ? null : resStdout);
```

Comment:
If I am not getting it wrong, what you are trying to do is the following: `return callback(null, resStdout || null);`

The || boolean operator returns the first operand if it is not falsy (this is 0 as an integer, "" as a string, false as a boolean). If the first operand is falsy, it returns the second one.